helm/panda/charts/server/sandbox/init-panda

```diff
@@ -1,16 +1,34 @@
 # install CA files
 /opt/panda/bin/panda_common-install_igtf_ca > /var/log/panda/install_igtf_ca.log 2>&1
 
-# fetch info from CRIC before running services
-tmpExe=`mktemp`
-cat <<EOT >> ${tmpExe}
-/opt/panda/usr/bin/panda_server-schedconfig_json
-/opt/panda/usr/bin/panda_server-configurator
-EOT
-chmod +x ${tmpExe}
+if [ -f /opt/panda/etc/cert/hostcert.pem ]; then
+    echo "Host certificate already mounted."
+else
+    echo "Host certificate not found. will generate a self-signed one."
+    unlink /etc/grid-security/hostkey.pem
+    unlink /etc/grid-security/hostcert.pem
+    mkdir /opt/panda/config
+    openssl req -x509 -nodes -days 365 -newkey rsa:2048 \
+        -subj "/C=US/DC=IDDS/OU=computers/CN=$(hostname -f)" \
+        -keyout /opt/panda/config/hostkey.pem \
+        -out /opt/panda/config/hostcert.pem
+    ln -fs /opt/panda/config/hostcert.pem /etc/grid-security/hostcert.pem
+    ln -fs /opt/panda/config/hostkey.pem /etc/grid-security/hostkey.pem
+fi
 
+#mkdir pandachache dir
```

Comment:
This part is required. Since /var/log/panda can be mounted from a pvc.
Before pandacache was under /var/cache/pandacache. I moved it to the pvc directory.